tomcat/src/main/java/org/apache/coyote/http11/response/HttpResponse.java

```diff
@@ -1,24 +1,38 @@
 package org.apache.coyote.http11.response;
 
+import org.apache.coyote.file.FileExtension;
+import org.apache.coyote.file.Resource;
 import org.apache.coyote.http11.HttpStatusCode;
 import org.apache.coyote.http11.header.Headers;
 import org.apache.coyote.http11.header.ResponseHeader;
+import org.apache.coyote.http11.path.Path;
+import org.apache.coyote.http11.version.HttpVersion;
+import util.BiValue;
+import util.StringUtil;
+
+import static org.apache.coyote.http11.response.Charset.UTF_8;
+import static org.apache.coyote.http11.response.ContentType.*;
 
 public class HttpResponse {
-    private final HttpStatusCode statusCode;
+    private HttpStatusCode statusCode;
     private final Headers headers;
-    private final String version;
-    private final byte[] body;
+    private HttpVersion version;
+    private byte[] body;
+
+    public HttpResponse() {
+        this.headers = new Headers();
+        this.body = new byte[0];
+    }
 
-    public HttpResponse(final HttpStatusCode statusCode, final Headers headers, final String version, final byte[] body) {
+    public HttpResponse(final HttpStatusCode statusCode, final Headers headers, final HttpVersion version, final byte[] body) {
         this.statusCode = statusCode;
         this.headers = headers;
         this.version = version;
         this.body = body;
     }
 
     public String getStatusLine() {
-        return String.join(" ", version, statusCode.getCodeString(), statusCode.getReasonPhrase()) + " ";
+        return String.join(" ", version.getVersion(), statusCode.getCodeString(), statusCode.getReasonPhrase()) + " ";
```

Comment:
```suggestion
        return "%s %s %s ".formatted(version.getVersion(), statusCode.getCodeString(), statusCode.getReasonPhrase());
```

join을 쓰게 되면 문자열 형태를 계산해야되니 format을 사용해보시죠

Reply:
지식이 늘었습니다! 🙇‍♂️